administrator/language/de-DE/de-DE.ini

```diff
@@ -75,12 +75,12 @@ JDETAILS="Details"
 JDISABLED="Deaktiviert"
 JENABLED="Aktiviert"
 JFALSE="Falsch"
-JFEATURED="Haupteintrag"
-JFEATURED_ASC="Haupteintrag aufsteigend"
-JFEATURED_DESC="Haupteintrag absteigend"
-JUNFEATURED="Kein Haupteintrag"
-JFEATURE="Haupteintrag"
-JUNFEATURE="Kein Haupteintrag"
+JFEATURED="Hauptbeitrag"
```

Comment:
Ich bin hier prinzipiell auf Deiner Seite, leider wird gerade „JFEATURED“ universell in Joomla! und vielen Erweiterungen benutzt und dann ist es kein „Hauptbeitrag“ mehr, sondern ein „Haupteintrag“. Daher die neutrale Fassung „Eintrag“.

Hier findet es sich z. B. im Joomla!-Core:
find . -type f -name "*.php" -o -name "*.xml" | xargs grep -sl "'JFEATURED'"
./libraries/cms/html/dropdown.php
./components/com_content/views/featured/view.feed.php
./administrator/templates/hathor/html/com_contact/contacts/default.php
./administrator/templates/hathor/html/com_content/articles/default.php
./administrator/components/com_contact/helpers/html/contact.php
./administrator/components/com_contact/views/contacts/tmpl/default.php
./administrator/components/com_contact/views/contacts/view.html.php
./administrator/components/com_content/views/articles/view.html.php

Und bei einem Kontakt kann man nicht von einem „Hauptbeitrag“ sprechen.